ui/components/tsconfig.build.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
```

Comment:
You can't extend from multiple tsconfig files, so erring on the side of importing from the local tsconfig and adding the build options (instead of making a global build options tsconfig and extending from that).